torch/_torch_docs.py

```diff
@@ -4660,30 +4660,9 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.log1p,
            r"""
-log1p(input, *, out=None) -> Tensor
 
-Returns a new tensor with the natural logarithm of (1 + :attr:`input`).
-
-.. math::
-    y_i = \log_{e} (x_i + 1)
-""" + r"""
-.. note:: This function is more accurate than :func:`torch.log` for small
-          values of :attr:`input`
-
-Args:
-    {input}
-
-Keyword args:
-    {out}
-
-Example::
-
-    >>> a = torch.randn(5)
-    >>> a
-    tensor([-1.0090, -0.9923,  1.0249, -0.5372,  0.2492])
-    >>> torch.log1p(a)
-    tensor([    nan, -4.8653,  0.7055, -0.7705,  0.2225])
-""".format(**common_args))
+Alias for :func:`torch.special.log1p`.
```

Comment:
Let's alias torch.special.log1p to torch.log1p for Python Array API consistency (see https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html?highlight=log1p#log1p-x).